projects/core/src/lib/operators/dialogs/raster-scaling/raster-scaling.component.html

```diff
@@ -0,0 +1,78 @@
+<geoengine-sidenav-header>Raster Scaling / Unscaling</geoengine-sidenav-header>
+
+<geoengine-dialog-help>
+    <p>
+        This operator applies scaling or unscaling to a raster data.
+        <br />
+        The raster scaling operator scales/unscales the values of a raster by a given scale factor and offset. This is done by applying the
+        formulas to every pixel.
+        <br />
+        For <b> unscaling </b> the formula is: <i>p_new = p_old * slope + offset</i>. <br />
+        For <b> scaling </b> the formula is: <i>p_new = (p_old - offset) / slope</i>. <br />
+        <i>p_old</i>> and <i>p_new</i> refer to the old and new pixel values. The slope and offset values are either properties attached to
+        the input raster or a fixed value. For more information see
+        <a href="https://docs.geoengine.io/operators/rasterunscaling.html">docs.geoengine.io</a>.
+    </p>
+</geoengine-dialog-help>
+
+<form [formGroup]="form" (ngSubmit)="add()" fxLayout="column" ngNativeValidate>
+    <div fxFlex>
+        <geoengine-layer-selection [types]="inputTypes" formControlName="layer"></geoengine-layer-selection>
+        <geoengine-dialog-section-heading title="Config" subtitle="Specify the output data type"></geoengine-dialog-section-heading>
+
+        <div formGroupName="scale">
+            <mat-slide-toggle formControlName="byKey" #byKey>Use scale factor from metadata key</mat-slide-toggle>
+            <mat-form-field appearance="fill">
+                <mat-label>Scale metadata <i>domain</i></mat-label>
```

Comment:
it would be cool to select the metadata instead of having to type it. How should the user know what metadata the raster has?

We would have to extend the RasterResultDescriptor though, or add another API to get the metadata